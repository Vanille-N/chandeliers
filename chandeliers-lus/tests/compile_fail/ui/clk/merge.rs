chandeliers_lus::decl! {
    node foo(b : bool; x, y : int) returns (i : int);
    let
        i = if b then x when b else y whenot b;
    tel;
}

fn main() {}
