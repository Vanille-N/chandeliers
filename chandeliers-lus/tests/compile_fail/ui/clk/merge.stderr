error: This expression is too slow: expected the implicit clock 'self, found 'when b
 --> tests/compile_fail/ui/clk/merge.rs:5:23
  |
5 |         i = if b then x when b else y whenot b;
  |                       ^^^^^^^^
  |
note: Found 'when b here
 --> tests/compile_fail/ui/clk/merge.rs:5:30
  |
5 |         i = if b then x when b else y whenot b;
  |                              ^
note: Expected because this expression moves at the implicit pace
 --> tests/compile_fail/ui/clk/merge.rs:5:16
  |
5 |         i = if b then x when b else y whenot b;
  |                ^
